src/main/groovy/com/bugsnag/android/gradle/BugsnagManifestTask.groovy

```diff
@@ -75,22 +77,24 @@ class BugsnagManifestTask extends BugsnagVariantOutputTask {
     }
 
     def shouldRun() {
-        def manifestPath = getManifestPath()
+        def shouldRun = false
+        def manifestPaths = getManifestPaths()
 
-        if (!manifestPath.exists()) {
-            return false
-        }
+        manifestPaths.each { manifestPath ->
+            if (!manifestPath.exists() || shouldRun) {
+                return
```

Comment:
This should return a boolean value

Reply:
As the return is within a closure, that will actually move onto the next element in the `each` loop rather than exit the method. I'll switch this (and the other loops added in this PR) to a `for` loop to make it clearer.